components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -7968,6 +7968,7 @@ public API getAPIbyUUID(String uuid, String organization) throws APIManagementEx
                 APIIdentifier apiIdentifier = api.getId();
                 apiIdentifier.setUuid(uuid);
                 api.setId(apiIdentifier);
+                api.setOrganization(organization);
```

Comment:
this is not needed since organization is populated in populateAPIInformation() method